content/python/concepts/random-module/terms/randint/randint.md

```diff
@@ -12,31 +12,31 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `random.randint()` function takes in two `int` values, and returns a pseudo-random integer within the defined range of `int` values.
+The `random.randint()` function returns a random integer that exists between two `int` values, `int_a` and `int_b` (inclusive), passed as arguments.
 
 ## Syntax
 
-```python
-random.randint(start, end)
+```pseudo
+random.randint(int_a, int_b)
 ```
 
-An alias for `randrange(start, end(+1))`.
+Another way of writing this would be `randrange(int_a, int_b + 1)`.
 
-## Example 1
+## Example
 
-Use `random.randint()` to return a random number between `0` and `50`:
+In the example below, `random.randint()` is used to return a random number between `0` and `50`:
 
-```codebyte/python
+```codebyte/py
```

Comment:
```suggestion
```py
```

Reply:
Doesn't seem a need for two codebyte examples.

Reply:
Good catch! Committing your suggestion.